css/style.css

```diff
@@ -14,4 +14,186 @@ a:hover{text-decoration: none;}
 ul li{list-style: none;}
 img{vertical-align: top;}
 h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight: inherit;}
-/*--------------------*/
+/*----------------------------------*/
+
+/*------- technical styles-------*/
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 18px;
+    line-height: 133%;
+    color: #ffffff;
+    background-color: #0d0d0d;
+    background: url("../img/bg.jpg") no-repeat center;
+    background-size: cover;
+}
+
+._container {
+    max-width: 1369px;
+    margin: 0 auto;
+}
+
+.wrapper {
+    min-height: 100%;
+    overflow: hidden;
+    display: flex;
+    flex-direction: column;
+}
+
+/* -------end of block------- */
+
+/* -------header,menu,register block------- */
+.header {
+    background-color: rgb(0, 0, 0, .8);
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 2;
+    padding: 0 20px;
+    display: flex;
+}
+.header__container {
+    min-height: 100px;
+    display: flex;
+    align-items: center;
+}
+
+.logo {
+    margin: 0 40px 0 0;
+}
+
+.logo img{
+    max-width: 90px;
+}
+.menu {}
+
+.menu__list {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.menu__first-block{
+    display: flex;
+}
+.menu__second-block{
+    display: flex;
+}
+.menu__link {
+    text-align: center;
+}
+.menu__item:not(:last-child) {
+    padding: 0 10px 0 0;
+}
+
+.menu__link {
+    color: #b3b3b3;
+    padding: 10px 15px;
+    font-family: 'Quicksand', sans-serif;
+    font-weight: 500;
+    transition: all .15s linear;
+}
+
+.menu__link:hover {
+    color: #ffffff
+}
+.menu__login {
+    margin: 0 0 0 543px;
```

Comment:
hardcoded pixels. Try to align the items by display flex or grid, instead of giving them fixed margins